ocs_ci/deployment/deployment.py

```diff
@@ -1610,8 +1632,26 @@ def deploy_with_external_mode(self):
         # enable secure connection mode for in-transit encryption
         if config.ENV_DATA.get("in_transit_encryption"):
             external_cluster.enable_secure_connection_mode()
-
-        setup_ceph_toolbox()
+        if config.DEPLOYMENT.get("multi_storagecluster"):
+            logger.info("not setting toolbox in multi-storagecluster")
+        else:
+            setup_ceph_toolbox()
+        logger.info("Checking ceph health for external cluster")
+        if not config.DEPLOYMENT.get("multi_storagecluster"):
+            try:
+                ceph_health_check(
+                    tries=30,
+                    delay=10,
+                )
+            except CephHealthException as ex:
+                raise CephHealthException(f"External ceph cluster not healthy - {ex}")
```

Comment:
```suggestion
                raise CephHealthException(f"External ceph cluster not healthy").with_traceback(ex.__traceback__)
```
I have found this solution here:
https://stackoverflow.com/questions/9157210/how-do-i-raise-the-same-exception-with-a-custom-message-in-python

WDYT?

The same for the line 1653

Reply:
Did quite understand your intentions here. What is wrong in the first place?

Reply:
We discussed offline we will do just:
```
except CephHealthException:
                raise CephHealthException(f"External ceph cluster not healthy")
                ```